plugins/feeds/public/bambenek_osint_ipmaster.py

```diff
@@ -0,0 +1,90 @@
+import re
+from datetime import timedelta
+
+from core import Feed
+import logging
+
+from core.observables import Hostname, Ip
+
+
+class Bambenek_osint_ipmaster(Feed):
+    default_values = {
+        "frequency":
+            timedelta(minutes=60),
+        "name":
+            "Bambenek_osint_ipmaster",
+        "source":
+            "http://osint.bambenekconsulting.com/feeds/c2-masterlist.txt",
+        "description":
+            "",
+    }
+    pattern = 'Master Indicator Feed for (\w+) non-sinkholed domains'
+    reg = re.compile(pattern)
+
+    def update(self):
+        for line in self.update_lines():
+            print(line)
+            self.analyze(line)
+
+    def analyze(self, line):
+        if not line or line[0].startswith("#"):
+            return
+
+        tokens = line.split(',')
+        c2_domain = []
+        ips_c2 = []
+        names_servers = []
+        ip_names_servers = []
+        context_feed = []
+        if len(tokens) == 6:
+            c2_domain = tokens[0]
+            ips_c2 = tokens[1].split('|')
+            names_servers = tokens[2].split('|')
+            ip_names_servers = tokens[3].split('|')
+            context_feed = tokens[4]
+
+            logging.debug(context_feed)
+            logging.debug(c2_domain)
+            logging.debug(ips_c2)
+            logging.debug(names_servers)
+
+            m = Bambenek_osint_ipmaster.reg.match(context_feed)
+            malware_family = ''
+            if m:
+                malware_family = m.group(1)
```

Comment:
I would extract this from `tokens[6]` since it seems less likely to change than the human-readable description. You can use a simpler regex like `r'/(\w+)\.txt$'`